agent/runtime-attach/src/test/resources/applicationinsights.json

```
@@ -0,0 +1,7 @@
+{
+  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://host.docker.internal:6060/",
```

just to be clear that nothing is listening there
```suggestion
  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://dummy",
```